docs/docs/features/split-keyboards.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Split Keyboards
+sidebar_label: Split Keyboards
+---
+
+ZMK supports split keyboard setups where two or more keyboard parts (also called "sides" or "halves"), each with their own controller running ZMK, communicate with each other to work as a single keyboard device.
```

Comment:
I had actually used "part" as the generic term, then added the sides/halves remark at the last minute because they are used throughout the rest of the docs. Let me know if you have a better term in mind than "part," I'll think about it as well.

Reply:
Perhaps "unit" or "node" might work nicely? We could also stick with part, but I'm worried about that being too generic.

Reply:
Both sound a bit too technical to me, so I think I prefer "parts" over them. I'd be interested to hear if anyone else has preferences.

Reply:
"pieces", perhaps?